### PR TITLE
[skdb] Default timezone to UTC

### PR DESCRIPTION
**Pull request by skiplabsdaniel**

### CODE REVIEW

File: sql/src/SqlDate.sk [hunk 1]
```diff
@@ -14,7 +14,7 @@ module SKDB;
  *  localtime       | adjusts time value to its left to the local time
  *  utc             | adjusts time value to its left to the utc time
  *  tz <timezone>   | adjusts time value to its left to the specified timezone (<timezone>: ±D | ±DD | ±DDDD | ±DD:DD | ±DD:DD:DD)
- *  isutc           | force the time value to its left to be in utc time ingnoring current timezome
+ *  islocal         | force the time value to its left to be in local time ingnoring current timezome
```
Comment (gregsexton) [inline sql/src/SqlDate.sk]: nit: ignoring

File: skdate/src/date.sk [hunk 2]
```diff
@@ -37,7 +37,7 @@ const daysInNonLeapYearMonth: Array<Int> = Array[
 class WeekDate(year: Int, week: Int, day: Day) uses Orderable {}
 
 base class Entity {
-  fun toCalendar(): Calendar;
+  fun toCalendar(tz: TZKind = TZLocal()): Calendar;
```
Comment (gregsexton) [inline skdate/src/date.sk:40]: Should the defaults not be utc for these params?

File: sql/src/SqlDate.sk [hunk 3]
```diff
@@ -226,7 +226,7 @@ fun strftime(
   timeValue: String,
   modifiers: Array<String>,
 ): String {
-  optDateTime = parseFullDate(context, timeValue, SKDate.TZLocal());
+  optDateTime = parseFullDate(context, timeValue, SKDate.utc);
```
Comment (gregsexton) [inline sql/src/SqlDate.sk:229]: This is the main change right?

**Comment by gregsexton** [comment]
Thanks for working on this. I think it will save a lot of future pain. Left some comments just to check my understanding.

File: sql/src/SqlDate.sk (hunk 3 above)
Comment (skiplabsdaniel) [reply sql/src/SqlDate.sk:229]: Yes

File: skdate/src/date.sk (hunk 2 above)
Comment (skiplabsdaniel) [reply skdate/src/date.sk:40]: It's the default of skdate library, date libraries used to be in local by default.

File: sql/src/SqlDate.sk [hunk 4]
```diff
@@ -14,7 +14,7 @@ module SKDB;
  *  localtime       | adjusts time value to its left to the local time
  *  utc             | adjusts time value to its left to the utc time
  *  tz <timezone>   | adjusts time value to its left to the specified timezone (<timezone>: ±D | ±DD | ±DDDD | ±DD:DD | ±DD:DD:DD)
- *  isutc           | force the time value to its left to be in utc time ingnoring current timezome
+ *  islocal         | force the time value to its left to be in local time ignoring current timezome
```
Comment (beauby) [inline sql/src/SqlDate.sk]:
```suggestion
 *  islocal         | force the time value to its left to be in local time
```

Comment (gregsexton) [reply sql/src/SqlDate.sk]: Lol, I was distracted by the typo and missed the bigger picture.

**Pull request merged**